Revit_oM/Properties/InstanceProperties.cs

```diff
@@ -21,15 +21,19 @@
  */
 
 using BH.oM.Base;
+using System.ComponentModel;
 
 namespace BH.oM.Adapters.Revit.Properties
 {
+    [Description("A generic wrapper BHoM type corresponding to any Revit family type. Name of InstanceProperties corresponds to family type name, type parameter values are stored in CustomData under corresponding keys.")]
```

Comment:
Might be out of scope for this, but could override the `Name` and `CustomData` properties in this class (they are all virtual on the base `BHoMObject` now, and move some of the descriptions there.